### DISCUSSION TITLE
Proi-UI website

### DISCUSSION THREAD

**Discussion by cgdougm** [specialdoom/proi-ui#18] [Ideas]
I can't seem to view [your website Proi-UI.com](https://proi-ui.com)

Wondering if you're rebuilding it, or if you thought of using [Storybook](https://storybook.js.org/docs/svelte/get-started/introduction)

**Comment by specialdoom** [comment]
I'm using the github pages now. And storybook is the next thing on my list. Thanks for checking in.

**Comment by specialdoom** [comment]
The site is up and running at https://specialdoom.github.io/proi-ui/.
